docs/vk-end-to-end-testing-guide.md

```
@@ -244,11 +244,11 @@ this test provides high confidence that:
 
 1. The `[dev-dependencies]` are correctly configured.
 
-2. Cargo's test runner can find and compile the `tests/e2e.rs` file.
+1. Cargo's test runner can find and compile the `tests/e2e.rs` file.
```

markdownlint-cli2 (0.21.0)

[warning] Ordered list item prefix
Expected: 2; Actual: 1; Style: 1/2/3

(MD029, ol-prefix)